### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -297,13 +297,20 @@ folder in the repository.
...
 
-```sh
-vercel
+Add `.vercel.json` in the project root directory with the following:
+
+```json
+{
+  "buildCommand": "npm run build --with-vercel",
+  "devCommand": "npm run start"
+}
```
Comment (dai-shi) [inline README.md:306]: Nice. Does it work?
We can omit `--with-vercel` as it detects from an env var.
I'm not sure how `devCommand` is used.
Don't we need "output directory"?

Comment (himself65) [reply README.md:306]: I'm still figuring it out. I think the best solution is vercel add waku commands into their document

Comment (dai-shi) [reply README.md:306]: Yeah, we can send them a PR. But, I hope to have a meantime solution too.

**Comment by dai-shi** [comment]
Let's reopen this if there's a progress.

**Pull request closed without merging**